index.html

```diff
@@ -1469,7 +1469,31 @@ <h4>@context</h4>
           </tbody>
         </table>
 
-
+        <p>
+The following values are acceptable values for the <code>@context</code> entry as
+a <a href="https://tools.ietf.org/html/rfc8259#section-7">JSON
+  String</a> or first item of a <a
+  href="https://tools.ietf.org/html/rfc8259#section-5">JSON Array</a>,
+represented as a <a href="https://tools.ietf.org/html/rfc8259#section-7">JSON
+  String</a>.
+        </p>
+        <table class="simple" style="width: 100%;">
+          <thead>
+            <tr>
+              <th>@Context Values</th>
```

Comment:
```suggestion
              <th>`@context` Values</th>
```
The `C` is never capitalized in use. It's not the first actual character, so it's debatable whether it should ever be capitalized in title case. Here, I think it's pretty certain that it should *NOT* be capitalized.

Reply:
No need for the ` since this is not markdown.

Reply:
@OR13 - As I was recently informed by @msporny, Respec turns the backticks into `<code>` wrappers. I prefer the `<code>` tags myself, but I'm going along with the Editor's expressed preference for fewer chars in the spec source.

Reply:
hmm, today I learned!

Reply:
Yeah, ReSpec understands Markdown. :)

Reply:
TIL as well. thanks!